matching-service/config/index.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     postgres: {
       options: {
         host: "localhost",
-        port: 5433,
+        port: process.env.PORT,
```

Comment:
Better to add `DB_` as the prefix for `process.env.PORT`.

Can help to update the `.env.sample` file also?